server/src/api/v2/events/get-events.ts

```diff
@@ -0,0 +1,67 @@
+import { Request, Response } from 'express';
+import { FindOptions} from 'sequelize';
+import { buildQuery, QueryBuilderConfig } from '../../../functions/query-builder.func';
+import { wrapResponse } from '../../../functions/response-wrapper';
+import { Event } from '../../../models/event.model';
+import { Vars } from '../../../vars';
+import { EventData } from '../../../interfaces/event.interface';
+import { User } from '../../../models/user.model';
+import { Organization } from '../../../models/organization.model';
+
+
+export async function getEvent(req: Request, res: Response): Promise<Response> {
+    let success = true;
+
+
+    const eventData: EventData | null = await Event
+        .scope({method: ['onlyCurrentOrg', Vars.currentOrganization.id]})
+        .findOne({
+            where: {
+                    id: req.params.id   
+            },
+            ... Vars.currentUser.is_admin ? {
+                include: [Organization, {model: User, as: 'author'}, {model: User, as: 'registered_users'}]
+            } : {
+                include: {
+                    model: User.scope('publicData')
+                }
```

Comment:
Das führt zu einem Fehler!
Error: Alias cannot be inferred: "Event" has multiple relations with "User"

Reply:
@DominikRuf Wo taucht der Fehler bei dir auf ? Bei mir erscheint nichts bzw. ich sehe ihn nirgends

Reply:
@LisaFremdt Wenn du als "nicht-admin" das getEvent /api/v2/events/:id machst kommt es zu dem Fehler.